app/services/message_sender.rb

```diff
@@ -30,7 +37,14 @@ def run
           error: error,
         )
       end
+    else
+      create_sent_scheduled_message(
+        employee: employee,
+        scheduled_message: message,
+        error: StandardError.new("Was unable to find a slack channel for this username"),
```

Comment:
oh this is an interesting way to surface this error. very nice! Wondering if we should be more specific with the error message. Something like: `"unable to find slack user with name #{employee.slack_username} and slack user id #{employee.slack_user_id}"` -- I think having that info in the error msg would be very helpful in debugging!